docs/config/index.md

```diff
@@ -44,6 +44,16 @@ export default defineConfig({
 })
 ```
 
+Starting with TypeScript 4.9, you can use the `satisfies` operator to ensure the config matches the `UserConfig` type, without jsdox annotations or the helper function import:
+
+```ts
+import type { UserConfig } from 'vite'
+
+export default {
+  // ...
+} satisfies UserConfig
+```
+
 Vite also directly supports TS config files. You can use `vite.config.ts` with the `defineConfig` helper as well.
```

Comment:
I think it's nice to add additional docs for satisfies. Perhaps it should be moved below the `Vite also directly supports TS config files` paragraph because this syntax relies on TypeScript. Something like this:
````suggestion
Vite also directly supports TS config files. You can use `vite.config.ts` with the `defineConfig` helper as well, or with the `satisfies` operator:

```ts
import type { UserConfig } from 'vite'

export default {
  // ...
} satisfies UserConfig
```
````

Reply:
Updated